plugins/faustwp/composer.json

```diff
@@ -1,37 +1,44 @@
 {
-    "name": "wpengine/faustwp",
-    "description": "Plugin for working with Faust.js, the Headless WordPress Framework.",
-    "type": "project",
-    "minimum-stability": "stable",
-    "require-dev": {
-        "codeception/codeception": "^4.1",
-        "codeception/module-asserts": "^1.0",
-        "codeception/module-cli": "^1.0",
-        "codeception/module-db": "^1.0",
-        "codeception/module-filesystem": "^1.0",
-        "codeception/module-phpbrowser": "^1.0",
-        "codeception/module-webdriver": "^1.0",
-        "codeception/util-universalframework": "^1.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-        "lucatume/wp-browser": "3.0.11",
-        "php-parallel-lint/php-parallel-lint": "^1.2",
-        "phpcompatibility/phpcompatibility-wp": "^2.1",
-        "phpunit/phpunit": "~7",
-        "roave/security-advisories": "dev-master",
-        "squizlabs/php_codesniffer": "^3.5.8",
-        "wp-coding-standards/wpcs": "^2.2",
-        "yoast/phpunit-polyfills": "^1.0"
-    },
-    "scripts": {
-      "lint": "parallel-lint -e php --no-colors --exclude vendor .",
-      "phpcs": "phpcs",
-      "phpcs:fix": "phpcbf",
-      "suite": ["@lint", "@phpcs", "@test"],
-      "test": "phpunit"
-    },
-    "config": {
-      "preferred-install": "dist",
-      "sort-packages": true,
-      "optimize-autoloader": true
+  "name": "wpengine/faustwp",
+  "description": "Plugin for working with Faust.js, the Headless WordPress Framework.",
+  "type": "project",
+  "minimum-stability": "stable",
+  "require-dev": {
+    "codeception/codeception": "^4.1",
+    "codeception/module-asserts": "^1.0",
+    "codeception/module-cli": "^1.0",
+    "codeception/module-db": "^1.0",
+    "codeception/module-filesystem": "^1.0",
+    "codeception/module-phpbrowser": "^1.0",
+    "codeception/module-webdriver": "^1.0",
+    "codeception/util-universalframework": "^1.0",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
+    "lucatume/wp-browser": "3.0.11",
+    "php-parallel-lint/php-parallel-lint": "^1.2",
+    "phpcompatibility/phpcompatibility-wp": "^2.1",
+    "phpunit/phpunit": "~7",
+    "roave/security-advisories": "dev-master",
+    "squizlabs/php_codesniffer": "^3.5.8",
+    "wp-coding-standards/wpcs": "^2.2",
+    "yoast/phpunit-polyfills": "^1.0"
+  },
+  "scripts": {
+    "lint": "parallel-lint -e php --no-colors --exclude vendor .",
+    "phpcs": "phpcs",
+    "phpcs:fix": "phpcbf",
+    "suite": [
+      "@lint",
+      "@phpcs",
+      "@test"
+    ],
+    "test": "phpunit"
+  },
+  "config": {
+    "preferred-install": "dist",
+    "sort-packages": true,
+    "optimize-autoloader": true,
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true
     }
```

Comment:
When running `composer phpcs`, I was getting prompted about [allow-plugins](https://getcomposer.org/doc/06-config.md#allow-plugins) (new security feature in composer 2.2). Declaring our allowed plugins silences the prompt without the need to add the `--no-interaction` flag when running the command.